### PR TITLE
docs: add an attention about dataset.HEAD to "Commit a Draft"

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2fzzmvg 为 Draft.commit 方法添加详细文档说明 commit 可能导致 branch 不同步](https://app.clickup.com/t/2fzzmvg)

File: docs/source/features/version_control/draft.rst [hunk 1]
```diff
@@ -109,6 +109,24 @@ saved into this commit.
    draft.commit(f"{COMMIT_TITLE}")
    draft.commit(f"{COMMIT_TITLE}", f"{COMMIT_DESCRIPTION}")
 
+.. attention::
+   SDK does not automatically update the ``dataset.HEAD`` after committing the draft. Therefore,
+   the dataset may be one commit behind the server. In addition, consider the concurrent operation
```
Comment (Lee-000) [inline docs/source/features/version_control/draft.rst:114]: Only one commit behind？Could there be more commits behind?
`consider the concurrent operation`? Rephrase this for better understanding?

File: docs/source/features/version_control/draft.rst [hunk 2]
```diff
@@ -109,6 +109,24 @@ saved into this commit.
...
+   of one team on one dataset.
+
+   It is necessary to ``checkout`` the specified branch, commit or tag before proceeding to the
```
Comment (Lee-000) [inline docs/source/features/version_control/draft.rst:117]: `checkout` or `checkout to`?

**Review by Lee-000 — CHANGES_REQUESTED** [review]

File: docs/source/features/version_control/draft.rst (hunk 2 above)
Comment (graczhual) [reply docs/source/features/version_control/draft.rst:117]: git use checkout

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**